extensions/markdown-language-features/src/features/previewContentProvider.ts

```diff
@@ -90,6 +90,18 @@ export class MarkdownContentProvider {
 			</html>`;
 	}
 
+	public async provideFileNotFoundContent(
+		resource: vscode.Uri,
+	): Promise<string> {
+		const resourcePath = path.basename(resource.fsPath);
+		return `<!DOCTYPE html>
+			<html>
+			<body class="vscode-body">
+				${resourcePath} cannot be found.
```

Comment:
Any user facing strings must be localizable.  For the body use `localize('preview.notFound', '{0} cannot be found', resourcePath)` so it can be translated

Reply:
@mjbvz , Ahh sorry for that , I missed it :) 
Fixed it now , Please check :)

Thanks!